applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt.c

```diff
@@ -288,6 +289,26 @@ static int random_static_addr_cfg(void)
 	return 0;
 }
 
+static int static_addr_random(void)
```

Comment:
It seems a bit strange that we have one function called `random_static_addr_cfg()` and another called `static_addr_random()`. I suggest the following: `random_static_addr_cfg` --> `ficr_static_addr_set` and `static_addr_random` --> `random_static_addr_set`